modules/digiTrustIdSystem.md

```diff
@@ -0,0 +1,74 @@
+## DigiTrust Universal Id Integration
+
+Setup
+-----
+The DigiTrust Id integration for Prebid may be used with or without the full
+DigiTrust library. This is an optional module that must be used in conjunction
+with the userId module.
+
+The full DigiTrust integration requires the DigiTrust package be included.
+DigiTrust may be initialized in the standard manner or through the params
+object. 
+
+You can use npm or reference the script on the cdn.
```

Comment:
suggest removing this stuff about referencing a script on the CDN, or using npm... let's make it as easy as possible, and no easier.

Reply:
OK

Reply:
Removed and added in more complete instructions from doc site.